src/components/organisms/WithNavigationBar/WithNavigationBar.tsx

```diff
@@ -32,7 +33,12 @@ export const WithNavigationBar: React.FC<WithNavigationBarProps> = ({
        *    works on the user side of things.
        */}
       <RelatedVenuesProvider venueId={venueId}>
-        <NavBar redirectionUrl={redirectionUrl} hasBackButton={hasBackButton} />
+        <EventsProvider venueId={venueId}>
+          <NavBar
+            redirectionUrl={redirectionUrl}
+            hasBackButton={hasBackButton}
+          />
+        </EventsProvider>
```

Comment:
@sunny-viktoryia @mike-lvov FYI: Not sure that this will be needed here, as I don't believe we show the schedule/etc in the admin pages, and everywhere else should be covered fine by `src/pages/VenuePage/TemplateWrapper.tsx`